aws/resource_aws_glue_workflow.go

```diff
@@ -71,20 +84,21 @@ func resourceAwsGlueWorkflowCreate(d *schema.ResourceData, meta interface{}) err
 
 func resourceAwsGlueWorkflowRead(d *schema.ResourceData, meta interface{}) error {
 	conn := meta.(*AWSClient).glueconn
+	ignoreTagsConfig := meta.(*AWSClient).IgnoreTagsConfig
 
 	input := &glue.GetWorkflowInput{
 		Name: aws.String(d.Id()),
 	}
 
-	log.Printf("[DEBUG] Reading Glue Workflow: %s", input)
+	log.Printf("[DEBUG] Reading Glue Workflow: %#v", input)
```

Comment:
I think we prefer `%s` as the AWS SDK generates `.String()` methods.

Reply:
I've used this notation previously (as i've seen in being used somewhere in the provider) and this was previously approved. so idk? i dont really mind which.

Reply:
No need to change then.